src/hotspot/share/gc/g1/g1EvacFailure.cpp

```diff
@@ -97,7 +97,7 @@ class RemoveSelfForwardPtrObjClosure {
       // explicitly and all objects in the CSet are considered
       // (implicitly) live. So, we won't mark them explicitly and
       // we'll leave them over NTAMS.
-      _cm->mark_in_next_bitmap(_worker_id, obj);
+      _cm->mark_in_next_bitmap_unconditionally(_worker_id, obj);
```

Comment:
I think this change, the introduction of this method, is unnecessary after moving the update to the nTAMS into the `G1EvacFailureRegions::record` method.